src/main/java/com/g4s8/ghman/bot/BotApp.java

```diff
@@ -94,7 +94,11 @@ public void run() {
                         new TkFork(
                             new FkCommand(
                                 "/notifications",
-                                    new TkNotifications(this.users)
+                                new TkNotifications(this.users)
+                            ),
+                            new FkCommand(
+                                "/reply",
```

Comment:
@victornoel In `TkNotifications` each notification is displayed as inline button, so as far as I understand the idea is that user clicks inline button with the notification he wants to answer, adds message, we process it and send to github. Otherwise how user is supposed to know `tid`? Now it is not displayed in notifications list... @g4s8 could you please clarify the idea for us?

Reply:
@olenagerasimova @victornoel yes, user get the list of notifications as inline buttons, each button has some data attached which is sent to web app on click, so when user clicks notification button we receive thread id as inline data and display the thread (send message back).
To reply to notification, user reply to telegram message with a thread, see #10:
> User should be able to reply to notification message in Telegram, bot should handle it and post a comment to Github ticket with user's message.

So no need to introduce new commands for reply, just use built in Telgram mechanisms like reply event.

Reply:
@olenagerasimova good point
@g4s8 I am not familiar with telegram and its API enough, what is the correct object to use to `use built in Telgram mechanisms like reply event.`?

Reply:
@g4s8 can we get some guidance here?

Reply:
@g4s8 ping

Reply:
@g4s8 can we get some guidance here

Reply:
@victornoel it can be checked by `reply_to_message` JSON field in update payload, Java SDK has same field in its DTO: https://github.com/rubenlagus/TelegramBots/blob/master/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/Message.java#L325